odh/base/monitoring/overrides/prometheus-operator/base/prometheus-rules.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
```

Comment:
I feel like we shouldn't have this label on here. Maybe better/more descriptive to use something like `prometheus: operate-first` or even just rely on `name: odh-monitoring`

Reply:
Yeah, I was thinking about something like that. :slightly_smiling_face:

Reply:
updated :+1: